pkg/flags/flags.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"flag"
+
+	"go.opentelemetry.io/collector/featuregate"
+)
+
+const (
+	featureGatesFlag = "feature-gates"
+)
+
+func Flags() *flag.FlagSet {
```

Comment:
missing docs? 


What is the purpose of this package? Shall we move other operator flags from main here?

Reply:
This is just for featureflags, i'll add some docs / rename this to show it's only for featureflags